legend-engine-xt-relationalStore-trino-execution-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProviderWithUserPasswordAuth_Trino.java

```diff
@@ -0,0 +1,95 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.plan.execution.stores.relational.connection.test;
+
+import org.finos.legend.engine.authentication.TrinoTestDatabaseAuthenticationFlowProvider;
+import org.finos.legend.engine.authentication.TrinoTestDatabaseAuthenticationFlowProviderConfiguration;
+import org.finos.legend.engine.plan.execution.stores.relational.config.TemporaryTestDbConfiguration;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.manager.ConnectionManagerSelector;
+import org.finos.legend.engine.plan.execution.stores.relational.connection.tests.api.dynamicTestConnections.TrinoTestContainersWithUserAndPasswordAuth;
+import org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.relational.connection.RelationalDatabaseConnection;
+import org.finos.legend.engine.shared.core.vault.EnvironmentVaultImplementation;
+import org.finos.legend.engine.shared.core.vault.PropertiesVaultImplementation;
+import org.finos.legend.engine.shared.core.vault.Vault;
+import org.finos.legend.engine.shared.core.vault.VaultImplementation;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.security.auth.Subject;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Properties;
+
+import static org.junit.Assume.assumeTrue;
+
+public class ExternalIntegration_TestConnectionAcquisitionWithFlowProviderWithUserPasswordAuth_Trino
+        extends RelationalConnectionTest
+{
+    private ConnectionManagerSelector connectionManagerSelector;
+    private VaultImplementation vaultImplementation;
+    public TrinoTestContainersWithUserAndPasswordAuth  trinoTestContainersWithUserAndPasswordAuth;
+
+    @BeforeClass
+    public static void setupTest()
+    {
+        Vault.INSTANCE.registerImplementation(new EnvironmentVaultImplementation());
+    }
+
+    @Before
+    public void setup()
+            throws IOException
+    {
+        startTrinoContainer();
+        TrinoTestDatabaseAuthenticationFlowProviderConfiguration flowProviderConfiguration = null;
+        TrinoTestDatabaseAuthenticationFlowProvider flowProvider = new TrinoTestDatabaseAuthenticationFlowProvider();
+        flowProvider.configure(new TrinoTestDatabaseAuthenticationFlowProviderConfiguration());
+        this.connectionManagerSelector = new ConnectionManagerSelector(new TemporaryTestDbConfiguration(-1), Collections.emptyList(), Optional.of(flowProvider));
+
+        Properties properties = new Properties();
+        properties.put("trino.user", "test");
+        properties.put("trino.password", "");
+
+        this.vaultImplementation = new PropertiesVaultImplementation(properties);
```

Comment:
why repeat the vault thing again, when it is already done as part of TrinoTestContainersWithUserAndPasswordAuth

Reply:
This is removed.